src/wp-includes/class-wp-theme-json.php

```diff
@@ -2379,7 +2379,7 @@ static function( $split_feature_selector ) use ( $clean_style_variation_selector
 		$pseudo_matches = array_values(
```

Comment:
This is a file controlled by gutenberg. This change will need to made in the gutenberg repo and ported across.

Reply:
I suppose that should be indicated in the file DocBlock.

Reply:
Note to self, Gutenberg file located at https://github.com/WordPress/gutenberg/blob/trunk/lib/class-wp-theme-json-data-gutenberg.php